backend/openapi.yaml

```diff
@@ -1032,20 +1031,875 @@ paths:
+             - minus sign (-) for descending order
+          required: false
+          schema:
+            type: array
+            default: [created_at]
+            items:
+              type: string
+              enum:
+                [
+                  created_at,
+                  +created_at,
+                  -created_at,
+                  updated_at,
+                  -updated_at,
+                  +updated_at,
+                ]
+
+      responses:
+        200:
+          description: Successfully retrieved a list of messages
+          content:
+            application/json:
+              schema:
+                type: object
+                required:
+                  - has_more
+                  - data
+                properties:
+                  has_more:
+                    type: boolean
+                  data:
+                    type: array
+                    items:
+                      $ref: '#/components/schemas/EmailMessageTransactional'
+        '400':
+          description: Bad Request
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/Error400Response'
+        '401':
+          description: Unauthenticated
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/Error401Response'
+        '500':
+          description: Internal Server Error
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/Error500Response'
+  /transactional/email/{emailId}:
+    get:
+      security:
+        - bearerAuth: []
+      tags:
+        - Email
+      summary: 'Get transactional email by ID'
+      parameters:
+        - in: path
+          name: emailId
+          required: true
+          schema:
+            type: string
+            example: 42
+      responses:
+        200:
+          description: Successfully retrieved transactional email with corresponding ID
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/EmailMessageTransactional'
+        '400':
+          description: Bad Request
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/Error400Response'
+        '401':
+          description: Unauthenticated
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/Error401Response'
+        '404':
+          description: Not Found
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/Error404Response'
+        '500':
+          description: Internal Server Error
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/Error500Response'
+
+  /campaign/{campaignId}/govsg:
```

Comment:
Qn: Why do we not attempt to reuse existing endpoints/schemas? At least for this endpoint, the response schemas only differ in one of the keys i.e. `email_templates` vs `govsg_templates`.

Reply:
In our code, we actually have a URL rewrite that fetch all the `campaign` types and then redirect to the corresponding `/<channel_type>` endpoint (for the other endpoints' purposes) hence this. We can keep in mind to refactor this but perhaps later

Reply:
@jia1 that's right